cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -102,8 +103,11 @@ type TestServerInstanceOptions struct {
 	// an apiserver version skew scenario where all apiservers use the same proxyCA to verify client connections.
 	ProxyCA *ProxyCA
 	// Set the BinaryVersion of server effective version.
-	// Default to 1.31
+	// If empty, binary version will be inferred from the build.
```

Comment:
```suggestion
	// If empty, effective version will default to version.DefaultKubeBinaryVersion.
```

(To encourage people that need to care to follow the breadcrumbs to the source-of-truth)